Tasks/XamariniOS/xamarinios.ts

```diff
@@ -5,29 +5,44 @@ import msbuildhelpers = require('msbuildhelpers/msbuildhelpers');
 
 import { ToolRunner } from 'vsts-task-lib/toolrunner';
 
-async function run() {
+function expandSolutionWildcardPatterns(solutionPattern: string): string {
+    const matchedSolutionFiles = tl.findMatch(null, solutionPattern, { followSymbolicLinks: false, followSpecifiedSymbolicLink: false });
+    tl.debug(`Found ${matchedSolutionFiles.length} solution files matching the pattern.`);
+
+    if (matchedSolutionFiles.length > 0) {
```

Comment:
Could matchedSolutionFiles be undefined or null? Check if (matchedSolutionFiles && matchedSolutionFiles.length ....

Reply:
It can't be null but I can add the check for durability.